.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint-staged
+exit 1
```

Comment:
Adding exit 1 in the end of this file will end every commit, even if all checks has passed
Witch brings the suspicion that your husky doesn't work